### ISSUE TITLE
EdgeDriver does not contain constructor with DriverService parameter

### ISSUE THREAD

**Issue by markloughery** [SeleniumHQ/selenium#8306]
## 💥 Regression Report

A clear and concise description of what the regression is.

## Last working Selenium version

Worked up to version: 3.14

Stopped working in version: 4.0.0-alpha-5

## To Reproduce

Detailed steps to reproduce the behavior:

- Create a ChromiumEdgeDriverService / EdgeDriverService object

- Create a EdgeDriver object and pass in the ChromiumEdgeDriverService object

```
ChromiumEdgeDriverService edgeDriverService = ChromiumEdgeDriverService.createDefaultService();
WebDriver driver = new EdgeDriver(edgeDriverService,new EdgeOptions());
```

## Expected behavior

There are no build errors

## Actual Behaviour

```
Build errors:
Error:(120, 18) java: no suitable constructor found for EdgeDriver(org.openqa.selenium.edge.EdgeDriverService,org.openqa.selenium.edge.EdgeOptions)
    constructor org.openqa.selenium.edge.EdgeDriver.EdgeDriver() is not applicable
      (actual and formal argument lists differ in length)
    constructor org.openqa.selenium.edge.EdgeDriver.EdgeDriver(org.openqa.selenium.edge.EdgeOptions) is not applicable
      (actual and formal argument lists differ in length)
    constructor org.openqa.selenium.edge.EdgeDriver.EdgeDriver(org.openqa.selenium.Capabilities) is not applicable
      (actual and formal argument lists differ in length)
```

## Environment

OS: Windows 10
Browser: Edge Chromium
Browser version: Version 81.0.416.72 (Official build) (64-bit)
Browser Driver version: EdgeDriver 81.0.416.72
Language Bindings version: Java 13.0.1

**Comment by wpyok500** [comment]
Sorry for borrowing and shipping

firefox OK，IE OK，Chrome  **NG** 
Error Code
![15](https://user-images.githubusercontent.com/14154547/81998537-59d26980-9685-11ea-878f-0fede5e26ad6.png)


**Environment**
OS: Windows 10
Browser: chrome
Browser version: Version 81.0.4044.138
Browser Driver version: ChromeDriver 81.0.4044.138
Language Bindings version: C#
Software : visual studio 2019
![11](https://user-images.githubusercontent.com/14154547/81998551-635bd180-9685-11ea-8699-562ba997d429.png)
![12](https://user-images.githubusercontent.com/14154547/81998564-66ef5880-9685-11ea-9ce4-bfa7d7d7828b.png)
![13](https://user-images.githubusercontent.com/14154547/81998568-6a82df80-9685-11ea-8b2c-7cfbff652e06.png)
![14](https://user-images.githubusercontent.com/14154547/81998572-6e166680-9685-11ea-8cfb-6acd22a34e1c.png)

**Comment by barancev** [comment]
Closed by 373d1fc608913915d5f8c24e5bfc6c0d17471e8e
